website/docs/modules/ROOT/partials/Publishing_Footer.adoc

```diff
@@ -1,4 +1,57 @@
-== Publishing to Sonatype Maven Central
+== `SonatypeCentralPublishModule` Configurations
+
+
+Below you are values you could override in the `SonatypeCentralPublishModule`.
+
+=== Module Settings
+Below are the default publishing settings on the module level, which can be explicitly configured by users like so:
+
+[source,scala]
+----
+object mymodule extends SonatypeCentralPublishModule {
+  override def sonatypeCentralGpgArgs: T[String] = "--batch, --yes, -a, -b"
+
+  override def sonatypeCentralConnectTimeout: T[Int] = 5000
+
+  override def sonatypeCentralReadTimeout: T[Int] = 60000
+
+  override def sonatypeCentralAwaitTimeout: T[Int] = 120 * 1000
+
+  override def sonatypeCentralShouldRelease: T[Boolean] = true
+  ...
+}
+----
+
+=== Argument Reference
+
+==== publishAll
+
+The `mill.scalalib/SonatypeCentralPublishModule/publishAll` method takes the following arguments:
+
+`username`: The username for calling the Sonatype Central publishing api. Defaults to the `SONATYPE_USERNAME` environment variable if unset. If neither the parameter nor the environment variable are set, an error will be thrown. +
+
+`password`: The password for calling the Sonatype Central publishing api. Defaults to the `SONATYPE_PASSWORD` environment variable if unset. If neither the parameter nor the environment variable are set, an error will be thrown. +
+
+`gpgArgs`: Arguments to pass to the gpg package for signing artifacts. Uses the `MILL_PGP_PASSPHRASE` environment variable if set. _Default: `[--passphrase=$MILL_PGP_PASSPHRASE], --no-tty, --pinentry-mode, loopback, --batch, --yes, -a, -b`._ +
+
+`publishArtifacts`: The command for generating all publishable artifacts (ex. `__.publishArtifacts`). Required. +
+
+`readTimeout`:  The timeout for receiving a response from Sonatype Central after the initial connection has occurred. _Default: 60000._ +
```

Comment:
This should be moved to the bottom of this section, since most people should not need to configure them.

You should also mention that  they can be set via command line flags and provide an example